Multisig/UI/App/UIKit/UI Library/LoadableViewController/LoadableViewController.swift

```diff
@@ -28,6 +28,7 @@ class LoadableViewController: UIViewController {
     @IBOutlet var allViews: [UIView]!
     var scrollViews: [UIScrollView] = []
     private var needsReload: Bool = false
+    let notificationCenter = NotificationCenter.default
```

Comment:
In my opinion, this should be a variable, not a constant, because then this dependency is changeable.